internal/factsengine/gatherers/corosyncconf_test.go

```diff
@@ -212,7 +212,7 @@ func (suite *CorosyncConfTestSuite) TestCorosyncConfFileNotExists() {
 	factsGathered, err := c.Gather(context.Background(), factsRequest)
 
 	expectedError := &entities.FactGatheringError{
-		Message: "error reading corosync.conf file: could not open corosync.conf file: " +
+		Message: "error reading corosync.conf file: could not open not_found: " +
```

Comment:
nitpick: Just with this output, it looks like we are not piping properly the error messages